docs/eBpfExtensions.md

```diff
@@ -73,7 +73,7 @@ functions that override the global helper functions provided by the eBPF runtime
 structure from provided data and context buffers.
 * `context_destroy`: Pointer to `ebpf_program_context_destroy_t` function that destroys a program type specific
 context structure and populates the returned data and context buffers.
-* `required_irql`: IRQL that the eBPF program runs at.
+* `maximum_irql`: The highest IRQL that the eBPF program can run at.
```

Comment:
```suggestion
* `maximum_irql`: The highest IRQL at which the eBPF program can be invoked.
```
For consistency with [ebpf_program_types.h](https://github.com/microsoft/ebpf-for-windows/pull/3149/files#diff-49fa4f21add58e3e7cee485187b8ec3aa7b84a480dcadd6d30792a78052a5998)